aif360/sklearn/detectors/detectors.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union
+
+from aif360.detectors import bias_scan
+from aif360.detectors.mdss import ScoringFunction
+
+import pandas as pd
+
+
+def bias_scan(
+    data: pd.DataFrame,
+    observations: pd.Series,
+    expectations: Union[pd.Series, pd.DataFrame] = None,
+    favorable_value: Union[str, float] = None,
```

Comment:
can we call these `X`, `y_true`, `y_pred`, and `pos_label`, respectively, for consistency with the sklearn API?

Reply:
Sure. I will make that change right away.